game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -143,10 +144,15 @@ class Cell(val id: Int, player: Player, var position: Vector2 = new Vector2(0f,
     None
   }
 
+  def defineAiState: AIState = {
```

Comment:
I think this could be simply named `aiState` if you like it, otherwise it can stay this way I won't mind.